### PR TITLE
feat: add hamburger

### PR DESCRIPTION
**Pull request by anik31**
Added hamburger menu in navbar

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
hi didnt find any issue in code but try to fix this in UI when I have opened the sidebar and I am trying to increase the viewport size it shows like this

Screenshot 2022-02-22 at 4 52 38 PM

File: components/navbar.css [hunk 1]
```diff
@@ -8,27 +8,139 @@
     flex-wrap: wrap;
     width: 100%;
     align-content: center;
-    justify-content: space-around;
+    justify-content: space-between;
     padding: 0.7rem 1rem;
+    /* position: fixed; */
+    /* top: 0; */
+    /* background-color: var(--white-color); */
```
Comment (Dev-AkshatGupta) [inline components/navbar.css]: Please try to remove all the unnecessary  code and commented code this makes it more readable

**Review by Dev-AkshatGupta — APPROVED** [review]
Else thing mentioned by Vineet everything else seems fine

**Comment by anik31** [comment]
> hi didnt find any issue in code but try to fix this in UI when I have opened the sidebar and I am trying to increase the viewport size it shows like this
> 
> Screenshot 2022-02-22 at 4 52 38 PM

It is beacause the default sidebar is also coming in the viewport. Will try to fix this. Thanks for reviewing

File: components/navbar.css (hunk 1 above)
Comment (anik31) [reply components/navbar.css]: Yes I will remove this. Thanks for reviewing

**Pull request merged**